controllers/articles.controllers.js

```diff
@@ -10,3 +13,15 @@ exports.getArticleById = (req, res, next) => {
 			next(err);
 		});
 };
+
+exports.patchArticleById = (req, res, next) => {
+	const articleId = req.params.article_id;
+	const incVotes = req.body.inc_votes;
+	updateArticleById(articleId, incVotes)
+		.then((updatedArticle) => {
+			res.status(200).send({ updatedArticle });
```

Comment:
Should still live in a key of article

Reply:
Ah, makes sense for future use - Thanks Andrea